src/core/bpf-firewall.c

```diff
@@ -342,6 +855,49 @@ static int bpf_firewall_count_access_items(IPAddressAccessItem *list, size_t *n_
         return 0;
 }
 
+static int bpf_firewall_add_port_range_access_items(
+                PortRangeAccessItem *list,
+                int map_fd,
+                int verdict) {
+
+        struct bpf_lpm_trie_key *key;
+        uint64_t value = verdict;
+        PortRangeAccessItem *a;
+        int r;
+        uint16_t port;
+        uint16_t protocol;
+
+        if (map_fd == 0)
```

Comment:
fds are invalid  < 0, and == 0 is actually a valid fd (though usually defined to map to stdin)